mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionProvider.scala

```diff
@@ -277,7 +277,10 @@ class CompletionProvider(
         case (_: Ident) :: (_: Import) :: _ =>
           mkWorkspaceItem(sym.fullNameBackticked)
         case _ =>
-          autoImports.editsForSymbol(sym) match
+          val forImportSymbol = v match
+            case w: CompletionValue.Workspace => w.applyOwner
+            case _ => sym
```

Comment:
Maybe it's enough to check it here? And not add a new field?
```suggestion
          val forImportSymbol =if (v.symbol.isMethod) v.symbol.owner else v.symbol
```
it would also work for expressions within string interpolation.

Reply:
Unfortunately we can't, because for example in `ListBuffer()`, owner of the `apply` method is trait IterableFactory. We have to somehow remember from which object this method came from

Reply:
I will see what we can do in case of string interpolation completions

Reply:
It seems suspicious that we are getting `apply` method symbols as Workspace completion.
 In theory, workspace completion can be created only from a type or companion object symbol as they comes from classpath search.

Reply:
I think that was added by me when I added apply completions